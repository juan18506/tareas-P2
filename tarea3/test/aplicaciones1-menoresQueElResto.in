crearListaPersonas

# creo una lista con personas y las siguientes edades: 22, 27, 25, 30, 45, 35
crearPersona 6 35 Persona6
insertarPersonaListaPersonas 1

crearPersona 5 45 Persona5
insertarPersonaListaPersonas 1

crearPersona 4 30 Persona4
insertarPersonaListaPersonas 1

crearPersona 3 25 Persona3
insertarPersonaListaPersonas 1

crearPersona 2 27 Persona2
insertarPersonaListaPersonas 1

crearPersona 1 22 Persona1
insertarPersonaListaPersonas 1

imprimirListaPersonas

# la pila de menores es la siguiente: 35, 30, 25, 22
menoresQueElResto

liberarListaPersonas

crearListaPersonas

# creo una lista con personas y las siguientes edades: 10, 8, 4, 3, 1
crearPersona 5 1 Persona5
insertarPersonaInicioListaPersonas

crearPersona 4 3 Persona4
insertarPersonaInicioListaPersonas

crearPersona 3 4 Persona3
insertarPersonaInicioListaPersonas

crearPersona 2 8 Persona2
insertarPersonaInicioListaPersonas

crearPersona 1 10 Persona1
insertarPersonaInicioListaPersonas

imprimirListaPersonas

# la pila de menores es la siguiente: 1
menoresQueElResto

liberarListaPersonas

crearListaPersonas

# creo una lista con personas y las siguientes edades: 1, 3, 8, 10, 20
crearPersona 5 20 Persona5
insertarPersonaInicioListaPersonas

crearPersona 4 10 Persona4
insertarPersonaInicioListaPersonas

crearPersona 3 8 Persona3
insertarPersonaInicioListaPersonas

crearPersona 2 3 Persona2
insertarPersonaInicioListaPersonas

crearPersona 1 1 Persona1
insertarPersonaInicioListaPersonas

imprimirListaPersonas

# la pila de menores es la siguiente: 20, 10, 8, 3, 1
menoresQueElResto

liberarListaPersonas

Fin