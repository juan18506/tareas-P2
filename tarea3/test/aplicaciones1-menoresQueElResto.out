1>2>#  creo una lista con personas y las siguientes edades: 22, 27, 25, 30, 45, 35.
3>4>5>6>7>8>9>10>11>12>13>14>15>Persona 1:  Persona1, 22 años
Persona 2:  Persona2, 27 años
Persona 3:  Persona3, 25 años
Persona 4:  Persona4, 30 años
Persona 5:  Persona5, 45 años
Persona 6:  Persona6, 35 años
16>#  la pila de menores es la siguiente: 35, 30, 25, 22.
17>Persona 6:  Persona6, 35 años
Persona 4:  Persona4, 30 años
Persona 3:  Persona3, 25 años
Persona 1:  Persona1, 22 años
18>19>20>#  creo una lista con personas y las siguientes edades: 10, 8, 4, 3, 1.
21>22>23>24>25>26>27>28>29>30>31>Persona 1:  Persona1, 10 años
Persona 2:  Persona2, 8 años
Persona 3:  Persona3, 4 años
Persona 4:  Persona4, 3 años
Persona 5:  Persona5, 1 años
32>#  la pila de menores es la siguiente: 1.
33>Persona 5:  Persona5, 1 años
34>35>36>#  creo una lista con personas y las siguientes edades: 1, 3, 8, 10, 20.
37>38>39>40>41>42>43>44>45>46>47>Persona 1:  Persona1, 1 años
Persona 2:  Persona2, 3 años
Persona 3:  Persona3, 8 años
Persona 4:  Persona4, 10 años
Persona 5:  Persona5, 20 años
48>#  la pila de menores es la siguiente: 20, 10, 8, 3, 1.
49>Persona 5:  Persona5, 20 años
Persona 4:  Persona4, 10 años
Persona 3:  Persona3, 8 años
Persona 2:  Persona2, 3 años
Persona 1:  Persona1, 1 años
50>51>Fin.
